### ISSUE TITLE
Suggestion for no-magic-numbers

### ISSUE THREAD

**Issue by fwh1990** [palantir/tslint#4412]
### Rule Suggestion

**Is your rule for a general problem or is it specific to your development style?**

 A general problem.

**What does your suggested rule do?**

`no-magic-numbers` should not report error when using in tsx attribute.

**List several examples where your rule could be used**

```typescript
class App extends React.PureComponent {
    render() {
        return <SomePage width={200} height={150} />;
    }
}
```
Because we both understand what the numbers mean.

**Additional context**

No.

**Comment by fwh1990** [comment]
So, you may add the new rule to repository `tslint-react`?

**Comment by JoshuaKGoldberg** [comment]
@fwh1990 this repository could take in a pull request to add the option to the `no-magic-numbers` rule that's off by default 😊

**Comment by fwh1990** [comment]
I appended option **ts.SyntaxKind.JsxAttribute** and **ts.SyntaxKind.JsxAttributes** to Rule.ALLOWED_NODES，but nothing changed. Can you add this enhancement yourself?

**Comment by adidahiya** [comment]
fixed by #4460
